src/silx/opencl/common.py

```diff
@@ -28,28 +28,18 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
+from builtins import property
 
 __author__ = "Jerome Kieffer"
 __contact__ = "Jerome.Kieffer@ESRF.eu"
 __license__ = "MIT"
 __copyright__ = "2012-2017 European Synchrotron Radiation Facility, Grenoble, France"
-__date__ = "09/09/2023"
+__date__ = "22/03/2024"
 __status__ = "stable"
-__all__ = [
-    "ocl",
-    "pyopencl",
-    "mf",
-    "release_cl_buffers",
-    "allocate_cl_buffers",
-    "measure_workgroup_size",
-    "kernel_workgroup_size",
-]
```

Comment:
Same, if `OpenCL` is lazy, everything else can be kept as before.

Reply:
maybe, but declaring it here ... removes the lazyness !

Reply:
OK, I see, it due to `if ocl.nb_devices == 0:` in `common.py`'s `def __getattr__(name):`

https://github.com/silx-kit/silx/pull/4093/files#diff-11a0e82f846b8e9f0ee348e05666f2f6c77b8927fa8dc0a42307112bc548f584R676